src/components/movies/MovieCard.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+    <div class="movie__card">
+        <h2 class="movie__title"> {{ movieTitle }}</h2>
+        <p class="movie__duration-time">{{ movieLength }}</p>
+        <img class="movie__poster" :src="moviePoster" />
+        <base-chip class="movie__category">
+            {{ movieCategory }}
+        </base-chip>
+    </div>
+</template>
+
+<script>
+import { formatMovieLength } from "@/helpers/dateHelper"
+export default {
+    props: {
+        movie: {
+            type: Object
+        }
+    },
+    computed: {
+        movieTitle() {
+            return this.movie?.title
+        },
+        movieLength() {
+            return formatMovieLength(this.movie?.length)
```

Comment:
I guess `formatMovieLength(null)` may not be handled so maybe this would be better here:

```suggestion
            return this.movie && formatMovieLength(this.movie.length)
```